### DISCUSSION TITLE
Multi-Target won't nugetize in release configuration

### DISCUSSION THREAD

**Discussion by EricSteuart** [devlooped/nugetizer#366] [Q&A] [answered]
Using the following multi-target solution, running `nugetize -p:configuration=Release` gives the following errors. If I run it without the configuration specified or even if I specify it as `Debug` it runs fine. Running `Release` against a non-multi-target solution doesn't have the problem.

![image](https://user-images.githubusercontent.com/78501046/227378404-27ed8ac2-1b3f-4951-b9ed-ede43692c49c.png)

[NugetizerTest - Multi-Target](https://github.com/devlooped/nugetizer/files/11035487/NugetizerTest.zip)

**Comment by kzu** [comment]
That looks like something new-ish from the .NET SDK which is trying to validate the output package which isn't being generated at all during nugetize. Creating an issue for it.

**Reply by kzu** [reply, marked as answer]
Got a fix, shipping soon.
